templates/dymaptic.GeoBlazor.Templates.csproj

```diff
@@ -3,7 +3,7 @@
 	<PropertyGroup>
 		<PackageId>dymaptic.GeoBlazor.Templates</PackageId>
 		<Title>GeoBlazor Project Templates</Title>
-		<PackageVersion>3.0.2-beta-2-beta-1</PackageVersion>
+		<PackageVersion>3.0.2-beta-11-beta-2-beta-1</PackageVersion>
```

Comment:
what do the -beta2-beta1 mean here?

Reply:
I have a powershell script that is updating the version in the templates, looks like it is broken. Good catch!